ansible/environments.yml

```diff
@@ -118,3 +118,33 @@
           NodeCount: 10
           AmiId: "ami-34a9364c" # AMI  ubuntu 16.04.3 with master user + python added.
           InstanceType: "m4.large"
+
+    - name: Ensure ae-uat-eucentral1 environment stack
+      cloudformation:
+        stack_name: "ae-uat"
```

Comment:
I see ae-uat used in 20+10+ (not in this PR) 10+10 nodes.
Are they connected automatically as peers in the config of the application running in UAT somehow? And what about running nodes?

Reply:
10 new in eu-central-1 10, new in ap-southeast-1, and 10 already created in us-west-2

Reply:
And 20 more at eu-east (OS Sofia)